docs/RFC/0004-ext2-filesystem-driver.md

```
@@ -0,0 +1,707 @@
+   (Workstream F).
+7. A conformance runner that ports pjdfstest into the QEMU integration suite
+   (Workstream G).
+
+## Design
+
+### Overview
+
+Seven workstreams. A, B, C are immediately startable (wave 1). D blocks on C.
+E blocks on D + A + B. F blocks on D. G blocks on E + F.
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)